packages/mobile/src/components/CurrencyDisplay.tsx

```diff
@@ -28,7 +29,8 @@ export default class CurrencyDisplay extends React.PureComponent<Props> {
       transform: [{ translateY: Math.round(size * 0.1) }],
     }
   }
-  amount() {
+
+  getAmount() {
     return this.props.amount == null ? '0.00' : getMoneyDisplayValue(this.props.amount || 0)
```

Comment:
amount can't be null anymore. This could be simplified to only `getMoneyDisplayValue(this.props.amount)`.